eng/CredScanSuppression.json

```diff
@@ -1,4 +1,83 @@
+                "temp2",
+                "123",
+                "Xabo2448",
+                "12NewPAwX0rd!",
+                "1234567890",
+                "((***Redacted***by***CredScan***))",
+                "proxyPassword",
+                "pass",
+                "dummyPassword",
+                "P@ssw0rd",
+                "12NewPA$$w0rd!",
+                "Samp1eP@ssw0rd"
+            ],
+            "_justification": "Secret used by test code, it is fake."
+        },
+        {
+            "file": [
+                "sdk\\cosmos\\azure-cosmos\\src\\test\\resources\\client.jks", 
+                "sdk\\spring\\azure-spring-boot\\src\test\resources\nopwdcert.pfx", 
+                "sdk\\spring\\azure-spring-boot\\src\test\resources\testkeyvault.pfx"
+            ],
+            "_justification": "Fake file used by test code."
+        },
+        {
+            "file": "sdk\\cosmos\\azure-cosmos\\src\\test\\resources\\server.jks",
+            "_justification": "Fake file used by test code."
+        },
+        {
+            "file": "sdk\\keyvault\\azure-security-keyvault-certificates\\src\\test\\resources\\pemCert.pem",
+            "_justification": "Fake file used by test code."
+        },
+        {
+            "file": "sdk\\keyvault\\microsoft-azure-keyvault-cryptography\\src\\test\\resources\\p256key.pem",
+            "_justification": "Fake file used by test code."
+        },
+        {
+            "file": "sdk\\keyvault\\microsoft-azure-keyvault-cryptography\\src\\test\\resources\\p256keynew.pem",
+            "_justification": "Fake file used by test code."
+        },
+        {
+            "file": "sdk\\keyvault\\microsoft-azure-keyvault-cryptography\\src\\test\\resources\\p384keynew.pem",
+            "_justification": "Fake file used by test code."
+        },
+        {
+            "file": "sdk\\keyvault\\microsoft-azure-keyvault-cryptography\\src\\test\\resources\\p521keynew.pem",
+            "_justification": "Fake file used by test code."
+        },
+        {
+            "file": "sdk\\keyvault\\microsoft-azure-keyvault-cryptography\\src\\test\\resources\\SECP256key.pem",
+            "_justification": "Fake file used by test code."
+        },
+        {
+            "file": "sdk\\keyvault\\microsoft-azure-keyvault-cryptography\\src\\test\\resources\\secp256keynew.pem",
+            "_justification": "Fake file used by test code."
+        },
+        {
+            "file": "sdk\\identity\\azure-identity\\src\\test\\resources\\cert-chain.pem",
+            "_justification": "Fake file used by test code."
+        },
+        {
+            "file": "sdk\\identity\\azure-identity\\src\\test\\resources\\key.pem",
+            "_justification": "Fake file used by test code."
+        },
+        {
+            "file": "sdk\\identity\\azure-identity\\src\\test\\resources\\keyStore.pfx",
+            "_justification": "Fake file used by test code."
+        },
+        {    
+            "hash": [
```

Comment:
As we chatted lets try to avoid adding hash's directly to the suppression file. Lets try to stick to placeholders and entire certificate files only.

Reply:
Right, focus on testing file suppresssion. Will remove